MobiFlight/Modifier/Comparison.cs

```diff
@@ -169,11 +164,6 @@ private string ExecuteStringComparison(ConnectorValue connectorValue)
             string result = connectorValue.String;
             string value = connectorValue.String;
 
-            if (!Active)
-            {
-                return connectorValue.String;
-            }
-
```

Comment:
Even though this isn't strictly related to the `Apply()` bug, I removed it anyway for consistency. The only place this method gets called in the entire codebase is from `Comparison.cs:Apply()` so it never had any real effect anyway.